apis/expenses-api-secured/tester.http

```diff
@@ -0,0 +1,32 @@
+// be sure to include scheme in urls (https://, http://)
+@api_url = FILL_ME_IN
+@issuer_base_url = FILL_ME_IN
+@issuer_client_id = FILL_ME_IN
+@issuer_client_secret = FILL_ME_IN
+
+# @name client_credentials
+POST {{issuer_base_url}}/oauth/token HTTP/1.1
+Content-Type: application/json
+
+{
+  "grant_type":"client_credentials",
+  "audience": "https://expenses-api",
+  "client_id" : "{{issuer_client_id}}",
+  "client_secret": "{{issuer_client_secret}}"
+}
+
+###
+@access_token = {{client_credentials.response.body.access_token}}
+
+### @name get_default_route
+GET {{api_url}}/ HTTP/1.1
+
+### @name get_total_route
+GET {{api_url}}/total HTTP/1.1
+
+### @name get_report_route_unsecured
+GET {{api_url}}/reports HTTP/1.1
+
+### @ name get_report_route_secured
+GET {{api_url}}/reports HTTP/1.1
```

Comment:
I get `401` when sending this request.

Reply:
The labs have you create the expenses api with a permission of `read:reports`.

Two ideas here:

1. Your API does not have this permission defined
2. Your test client is not authorized for the given permission

![Screen Shot 2020-10-08 at 12 48 16 PM](https://user-images.githubusercontent.com/73120/95506287-8e143280-0964-11eb-875c-bac3894323ca.png)

![Screen Shot 2020-10-08 at 12 49 51 PM](https://user-images.githubusercontent.com/73120/95506498-d2073780-0964-11eb-8500-2a96ae502201.png)

Reply:
Do you think the readme should mention required scopes?

Reply:
Definitely should mention the scope (in case someone does this lab without the lab instructions)

I just tried the scope and I still get the audience error when spending the AT

Reply:
Please do not spend your access tokens they may only be redeemed by the service they are intended for.

Reply:
I added a scopes table for each api.